### PR TITLE
Suraj

### LINKED ISSUE #11: Statistics Page
Showing the number of Registrations, Institutions participating, Sponsors, Partners, etc.

### PR DESCRIPTION
**Pull request by Skb08**
**Description**

This PR fixes #11 

**Notes for Reviewers**

### CODE REVIEW

**Review by duckling69 — APPROVED** [review]
please just change names of css classes to be more specific and remove unwanted comments if we don't need them 
LGTM apart from that good work 👏

File: src/app/page.js [hunk 1]
```diff
@@ -60,5 +63,9 @@ export default function Home() {
         </a>
       </div>
     </main>
+      <div className="flex min-h-screen min-w-screen flex-col items-center  xl:p-0 md:text-2xl bg-black shadow"  >
```
Comment (8rxn) [inline src/app/page.js]: Needs a little more padding from top and takes up height more than it needs
image

File: src/app/suraj/Statistics.js [hunk 2]
```diff
@@ -0,0 +1,78 @@
+import React from 'react'
+import './statisticsStyle.css'
+
+
+const Statistics = () => {
```
Comment (8rxn) [inline src/app/suraj/Statistics.js]: Move this file into the appropriate folder as informed in group chat.

File: public/Images/base.svg
Comment (8rxn) [inline public/Images/base.svg:1]: Use relevant filenames instead of vague ones like "base". Also prepend them with their target component: Sponsor here.

**Review by 8rxn — CHANGES_REQUESTED** [review]
@Skb08 Please Make these minor changes and it'll be ready to merge.

**Comment by duckling69** [comment]
Thanks @Skb08 @8rxn 👏

**Pull request merged**